db/sqloffload.c

```diff
@@ -189,63 +189,43 @@ void osql_cleanup(void)
     bdb_osql_destroy(&bdberr);
 }
 
-char *osql_breq2a(int op)
+const char *osql_tp_str[] = {
```

Comment:
What do you think about doing it like: `CLIENT_TYPE_TO_STR`?

Reply:
But simpler still, so you just quote the field using preprocessor

Reply:
I'll fix this in a subsequent PR. Good suggestion.